Lesson27/Merve/create.js

```diff
@@ -0,0 +1,24 @@
+document.getElementById('newForm').addEventListener('submit', function (event) {
+    event.preventDefault();
+
+    const title = document.getElementById('title').value;
+    const content = document.getElementById('content').value;
+
+
+    createPost(title, content);
+});
+
+function createPost(title, content) {
+
+    const postHtml = `
+        <div class="post">
+            <h3>${title}</h3>
+            <p>${content}</p>
+        </div>
+    `;
+
+    const formElement = document.getElementById('newForm');
+    formElement.insertAdjacentHTML('afterend', postHtml);
```

Comment:
Don't insert a div element into a form. Instead, add a div element on the create-post.html page and append the title and content to it.

Also, you forget to do the POST HTTP request here.